monorepo/services/server/src/graphql/definitions/campaign.js

```diff
@@ -68,6 +68,8 @@ type Campaign {
   deleted: Boolean!
   createdAt: Date
   updatedAt: Date
+  showAdvertiserCTOR: Boolean!
+  showTotalAdClicksPerDay: Boolean!
```

Comment:
These fields can be required when creating/updating the model, but they can't be marked as non-nullable here, because none of the existing campaigns will have the fields set. I would just drop the `!` here, otherwise you would have to add logic to the resolver to ensure a boolean value is returned.

If you load a campaign without these settings, this should cause an error in the UI.